main.go

```diff
@@ -175,13 +175,11 @@ var initCmd = &cli.Command{
 			return err
 		}
 
-		// create the package directory with a temporary file so that go recognises it as a package
-		// and autobinding doesn't error out
-		tmpPackageNameFile := "graph/model/_tmp_gqlgen_init.go"
-		if err := initFile(tmpPackageNameFile, "package model"); err != nil {
+		// create initial file for initial autobind
+		modelFilename := "graph/model/doc.go"
+		if err := initFile(modelFilename, "package model"); err != nil {
```

Comment:
@mtibben Is it easy to understand? If you still don't understand this, you're the one who needs to fix it. Because I am under no obligation to fix it